addons/web/static/src/js/core/session.js

```diff
@@ -272,6 +273,7 @@ var Session = core.Class.extend(mixins.EventDispatcherMixin, {
     },
     get_file: function (options) {
         options.session = this;
+        options.context = this.user_context
```

Comment:
should not be needed thanks to https://github.com/odoo/odoo/commit/2cc360b61889e4c2831c815e1f4358bcd75ccff3#diff-2c418cb78b7c562a79e7032645a537d2R876